torch/distributed/distributed_c10d.py

```diff
@@ -1714,6 +1714,8 @@ def broadcast_object_list(object_list, src=0, group=None):
         src (int): Source rank from which to broadcast ``object_list``.
         group: (ProcessGroup, optional): The process group to work on. If None,
             the default process group will be used. Default is ``None``.
+        device (``torch.device``, optional): device to send from or receive
+            to (default: ``torch.device("cpu")``).
```

Comment:
Shall we also explain the default behavior? The docstring says it defaults to CPU, but the code seems suggest that it will use current device if the backend is NCCL?

Reply:
Good point, updated. Was trying to minimize the change since I'm not familiar with the codebase yet :)